2.yaml

```diff
@@ -0,0 +1,11 @@
+Parameters:
+        UnusedParameter:
+                Type: String
+
+        AnotherOne:
```

Comment:
cfn-lint: W2001 Parameter AnotherOne not used.  <!-- Automatically posted by Lintly -->